cloudify_rest_client/snapshots.py

```diff
@@ -112,11 +112,14 @@ def create(self,
                include_credentials,
                include_logs=True,
                include_events=True,
-               queue=False):
+               queue=False,
+               include_metrics=None):
         """
         Creates a new snapshot.
 
         :param snapshot_id: Snapshot id of the snapshot that will be created.
+        :param include_metrics: Deprecated parameter, kept for old client
```

Comment:
Just write that's deprecated and should not be used and write the unit tests part on the commit